sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md

```diff
@@ -1,12 +1,31 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2020-10-30)
+
+### New Features
 
 - Regenerate protocol layer from service API version 2020-10-31
 - Update service API version to use service API version 2020-10-31 by default
-- Add optional parameters for TraceParent and TraceState to all service request APIs to support distributed tracing
+
+
+### Breaking changes
```

Comment:
Might deserve a note that these are breaking changes from preview, and that normally from GA version to version we would not have them. These are more like advisory changes for those who used the preview package. Thoughts?

Reply:
Sure, it wouldn't hurt